lib/fastlane/plugin/ddg_apple_automation/helper/git_helper.rb

```diff
@@ -77,6 +77,31 @@ def self.assert_branch_has_changes(release_branch)
 
         changed_files.any?
       end
+
+      def self.latest_release(repo_name, prerelease, platform = nil, github_token)
+        client = Octokit::Client.new(access_token: github_token)
+
+        current_page = 1
+        page_size = 25
+
+        loop do
+          releases = client.releases(repo_name, per_page: page_size, page: current_page)
+          break if releases.empty?
+
+          # If `prerelease` is true, return the latest release that matches the platform regardless of whether it's public.
+          # If `prerelease` is false, then ensure that the release is public.
+          matching_release = releases.find do |release|
+            (prerelease || !release.prerelease) && (platform.nil? || release.tag_name.end_with?("+#{platform}"))
```

Comment:
I'm assuming we're fine to check for `release.prelease` as a way to determine whether a tag is public or internal. It was either this or using a regex to check the tag format, and this feels simpler, providing that it's an assumption we can rely on.